cfme/tests/services/test_dialog_element_in_catalog.py

```diff
@@ -55,7 +57,7 @@ def service_dialog(appliance, widget_name):
 
 
 @pytest.fixture(scope="function")
-def catalog_item(appliance, service_dialog, catalog):
+def catalog_item_local(appliance, service_dialog, catalog):
```

Comment:
You actually don't need to change the name of the fixture, pytest will resolve the most local one matching the node's scope.

Reply:
Actually `pytest` took local `catalog_item` fixture  instate of `catalog_item` global fixture; when we call global `service_vm`.